tests/rptest/scale_tests/many_partitions_test.py

```diff
@@ -518,7 +536,7 @@ def _single_node_restart(self, scale: ScaleParameters, topic_names: list,
         # Wait for leaderships to stabilize on the surviving nodes
         wait_until(
             lambda: self._node_leadership_evacuated(topic_names, n_partitions,
-                                                    node_id), 30, 1)
+                                                    node_id), 30, 5)
```

Comment:
Is this really needed? 5 minutes is a long time.

Reply:
It is the backoff_sec param. And, I set it to 5s to log less often.

```
:param backoff_sec: number of seconds to back off between each failure to meet the condition before checking again
```